pkg/syncer/metadatasyncer.go

```diff
@@ -142,6 +142,19 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		return err
 	}
 	metadataSyncer.clusterFlavor = clusterFlavor
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorVanilla ||
```

Comment:
Can we move line 145 to 154 to original place in the else case of `if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorGuest {`

Reply:
Done